Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+
+#include "Person.h"
+
+
+Person::Person(){
+  age_ = new int (0);
```

Comment:
As a cool tip you can change this to (through the use of _initialization lists_):
`Person::Person(): age_(new int){}`